sky_provider/operators/sky_task.py

```diff
@@ -0,0 +1,260 @@
+from typing import Dict, Optional, Sequence
+
+from airflow import __version__ as airflow_version
+from airflow.exceptions import AirflowException
+from airflow.providers.standard.operators.python import \
+    PythonVirtualenvOperator
+from packaging.version import Version
+
+if Version(airflow_version) >= Version("3.0.0"):
+    from airflow.sdk import Variable
+else:
+    from airflow.models import Variable
+
+from .credentials_utils import (CREDENTIAL_HANDLERS, ClusterCredentials,
+                                CredentialsOverride)
+
+
+class SkyTaskOperator(PythonVirtualenvOperator):
```

Comment:
we may want to do a quick poll for the operator names:
1. SkyTaskOperator
2. SkyPilotClusterJobOperator
3. SkyPilotJobOperator

Reply:
Discussed offline, we agreed on `SkyPilotClusterOperator`.

In that case, should we rename the package too then to `skypilot_provider`?

Reply:
`skypilot_provider` sounds good to me

Reply:
Sounds good, I've changed it